app/helpers/days_helper.rb

```diff
@@ -19,4 +19,13 @@ def day_status_css(d)
   def feedback_submitted?(day)
     student? && current_user.day_feedbacks.find_by(day: day.to_s)
   end
+
+  def total_cohort_students
+    @total_cohort_students ||= cohort.students.length
```

Comment:
It turns out that the students aren't already loaded. This would be more correctly written using `count` after all.

```ruby
@total_cohort_students ||= cohort.students.count
```